clients/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface UserOperation {
```

Comment:
Since we get typescript types based on our solidity, we could use the generated [UserOperationStruct](https://github.com/statechannels/nitro-smart-contract-wallet/blob/4aaf4d73b544a9f50379d13a5d05e450dba57a3b/test/UserOp.ts#L12)

Reply:
These are the versions I am using:
```
node.js: v18.16.0
yarn: v1.22.19
```